src/app/DTT/upload/dropzone.js

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react'
+import { useDropzone } from 'react-dropzone'
+import { Box, Flex, Text, Input, Button, Center, Image } from '@chakra-ui/react'
+import { isEmpty, map } from 'ramda'
+
+const DropZone = ({ handleSave, closeDropzone, maxSize = '5000000' }) => {
+  const [files, setFiles] = useState([])
+
+  const { getRootProps, getInputProps } = useDropzone({
+    accept: 'image/*',
+    maxSize: maxSize,
+    onDrop: (acceptedFiles, rejectedFiles) => {
+      setFiles(
+        acceptedFiles.map(file =>
+          Object.assign(file, {
+            preview: URL.createObjectURL(file),
+          }),
+        ),
+        rejectedFiles.map(file => alert('There was an error, try again!')),
```

Comment:
can we change the alert to a toast notification error

Reply:
Done.